packages/contracts/contracts/core/SphinxModule.sol

```diff
@@ -377,90 +269,31 @@ contract SphinxModule is ReentrancyGuard, Enum, ISphinxModule, Initializable {
                 uint256 gas,
                 bytes memory txData,
                 Enum.Operation operation,
-                bool requireSuccess
+                bool requireIsContract
             ) = abi.decode(leaf.data, (address, uint256, uint256, bytes, Enum.Operation, bool));
 
+            // If `requireIsContract` is `true`, check that the `to` field is a contract.
+            require(!requireIsContract || to.code.length > 0, "TODO(docs)");
```

Comment:
Minor comment: 
I expect an auditor would recommend checking the code hash instead of just checking the length. We know we don't want to do this because it may cause incompatibilities on some zk rollups. So we'll probably want to document this somewhere.